README.rst

```diff
@@ -14,11 +14,11 @@ API
 
 Basically Beniget provides three analyse:
 
-- ``beniget.Ancestors`` that maps each node to the list of enclosing nodes;
-- ``beniget.DefUseChains`` that maps each node to the list of definition points in that node;
-- ``beniget.UseDefChains`` that maps each node to the list of possible definition of that node.
+- `beniget.Ancestors` that maps each node to the list of enclosing nodes;
```

Comment:
mmmh, that's not how rst is higlighting code, why would you want to use single quotes?